docs/guides/proofreading.md

```diff
@@ -30,49 +53,51 @@ Once SLEAP has measured the similarity between all the candidates and the instan
+4) Check the **Cull to Target Instance Count** checkbox
 
-More training data?
---------------------
+{{ set_instance_count }}
+
+Once you have the desired number of instances in every frame, SLEAP connects identities with a simple heuristic: if exactly one track identity was dropped from frame N and exactly one new track identity was added in frame N+1, it matches up the dropped and the new tracks.
 
-Often your models will fail to predict *all* of the instances on *all* of the frames. Even if you're happy enough with the result since you can interpolate missing data, it's possible that the missing instances will cause problems when we try to determine track identities across frames, so if your tracking results are poor, you may wish to :ref:`merging`.
+## More training data?
 
-The "track cleaning" script
-----------------------------
+Often your models will fail to predict *all* of the instances on *all* of the frames. Even if you're happy enough with the result since you can interpolate missing data, it's possible that the missing instances will cause problems when we try to determine track identities across frames, so if your tracking results are poor, you may wish to {ref}`merging`.
 
-There's an experimental command-line utility which tries to match up lost identities. You need to give it a predictions file which already has track assignments, and specify how many instances there should be. It looks for frames where there's exactly one lost identity and exactly one newly spawned identity, and it joins these into a single track. Suppose you have a predictions file at :code:`path/to/predictions.h5` and you want to end up with three distinct tracks. You can run
+## The "track cleaning" script
 
-::
+There's an experimental command-line utility which tries to match up lost identities. You need to give it a predictions file which already has track assignments, and specify how many instances there should be. It looks for frames where there's exactly one lost identity and exactly one newly spawned identity, and it joins these into a single track. Suppose you have a predictions file at {code}`path/to/predictions.h5` and you want to end up with three distinct tracks. You can run
 
-  python -m sleap.info.trackcleaner path/to/predictions.h5 -c 3
+```
+python -m sleap.info.trackcleaner path/to/predictions.h5 -c 3
+```
 
-This will result in a new file at :code:`path/to/predictions.cleaned.h5`. This file has the same format as the SLEAP labels and predictions files.
+This will result in a new file at {code}`path/to/predictions.cleaned.h5`. This file has the same format as the SLEAP labels and predictions files.
 
 The main worry is that this script will connect identities which should be distinct, so that in place of **lost** identities you'll now have more **mistaken** identities, which can be harder to locate when proofreading. Tools and techniques for finding **mistaken** identities during proofreading are explained below.
```

Comment:
This seems obsolete. This functionality is already tied into the GUI in the Inference Pipeline under **Post-tracker data cleaning** *Connect Single Track Breaks*.

![image](https://user-images.githubusercontent.com/38435167/170384669-544cc6ba-20f8-4708-b153-877e2cf450a5.png)